flink-yarn/pom.xml

```diff
@@ -99,6 +99,13 @@ under the License.
 		</dependency>
 
 		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-s3-fs-hadoop</artifactId>
+			<version>${project.version}</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
We could move the S3 upload test to `flink-yarn-tests`. That way we would only add this dependency to `flink-yarn-tests`.

Reply:
We could...
I wasn't putting this test there because in the project's pom.xml it says `There is a separate "flink-yarn-tests" package that expects the "flink-dist" package to be build before.` which we do not need for this test.

whatever you prefer...

Reply:
I think it's also ok this way :-)